### PR TITLE
fix: ability to select 4 tags in project creation

### PR DESCRIPTION
**Pull request by tortcherx**

### CODE REVIEW

File: src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx [hunk 1]
```diff
@@ -162,19 +162,19 @@ export const ProjectTagsCreateEdit = ({
...
+  const disableShowAddTag = inputValue.length < 4 || createLoading
 
   return (
     <FieldContainer
       title="Tags"
       subtitle={
         <span>
           {t(
-            'Get discovered more easily by users through Tags. You can select up to 3 project tags.',
+            'Get discovered more easily by users through Tags. You can select up to 4 project tags.',
```
Comment (sajald77) [inline src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx:177]: please update this text in the english.json file as well, that'll helps us track the updated translations as well.

File: src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx [hunk 2]
```diff
@@ -162,19 +162,19 @@ export const ProjectTagsCreateEdit = ({
     )
   }
 
-  const isDisabled = tags.length >= 3
+  const isDisabled = tags.length >= 4
   const showAddTag = !tagOptions.some((tag) =>
     tag.label.toLowerCase().includes(inputValue.toLowerCase()),
   )
-  const disableShowAddTag = inputValue.length < 3 || createLoading
+  const disableShowAddTag = inputValue.length < 4 || createLoading
```
Comment (sajald77) [inline src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx:169]: maybe add a const MAX_TAGS_ALLOWED, value, that'll be easy to see and use. We use all caps for simple constant cases like this one.

File: src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx (hunk 1 above)
Comment (travishoglund) [reply src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx]: updated

File: src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx (hunk 2 above)
Comment (travishoglund) [reply src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx]: Updated

**Review by sajald77 — APPROVED** [review]
Looks great.

**Pull request merged**